source/jormungandr/jormungandr/parking_space_availability/car/common_car_park_provider.py

```diff
@@ -73,9 +83,32 @@ def get_informations(self, poi):
         if data:
             return self.process_data(data, poi)
 
-    def support_poi(self, poi):
+    def _is_poi_coords_within_shape(self, poi):
+        if self.shape is None:
+            return True
+
+        try:
+            coord = poi['coord']
+            coords = [float(coord['lon']), float(coord['lat'])]
+            return self.shape.contains(shapely.geometry.Point(coords))
+        except KeyError as e:
```

Comment:
Maybe adding an ` except ValueError` if members are not float type

Reply:
This will be caught further down within the generic `except Exception as e:`